### DISCUSSION TITLE
[Feature] Search by image location or date-taken in metadata

### DISCUSSION THREAD

**Discussion by jmluxem** [immich-app/immich#7246] [Feature Request]
### The feature

I commonly finding myself searching for images based on the location the photo was taken in Apple Photos - whether it be by general City, State, or by the precise street address known. Immich has the Map display which does help quite a bit as feature-parity, but being able to search based on photo locations would be a big help.
Similarly, being able to search for photos from a specific date does not appear to work currently but would be a great feature to have. Both of these are related to photo metadata which is why I've included both. 

If this is already possible and I'm just missing something, please do let me know!

Thank you!!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
New search is work in progress with these features. Preview: 
<img width="1075" alt="301252946-1f15c3de-c049-4b43-9ed2-5b05b0927d5b" src="https://github.com/immich-app/immich/assets/5739463/163f8b31-df65-460e-aed1-84286f3d981a">

**Reply by jmluxem** [reply]
Oh awesome! Thanks for the info!

**Comment by Psy185** [comment]
Is it possible to search for photos without any location? I would like to manually set photos location when it's missing.

**Reply by schuhbacca** [reply]
I don't know if there's a way to search for places excluding a location other than doing a DB query

**Reply by Psy185** [reply]
What I now did was importing everything in Lightroom (oof) at the existing directory and there I can filter for missing GPS data. But entering GPS data is a bit inconvenient there because you can only enter the coordinates. A map chooser would be nicer :)

But better than nothing.
